packages/datagrid/src/components/DefaultCellRenderer/DefaultCellQualityIndicator.component.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import * as RuleCore from '@talend/rule-core';
+
+const { QualityDetailsIndicator } = RuleCore.components;
```

Comment:
I am curious, why dont you only import the targeted component?

Reply:
TRR API doesn't allow it